src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -51,6 +51,10 @@ static ucs_config_field_t uct_gdr_copy_md_config_table[] = {
     {"MEM_REG_GROWTH", "0.06ns", "Memory registration growth rate", /* TODO take default from device */
      ucs_offsetof(uct_gdr_copy_md_config_t, uc_reg_cost.c), UCS_CONFIG_TYPE_TIME},
 
+    {"RCACHE_OVERHEAD", "250ns", "gdr_copy registration cache lookup overhead",
```

Comment:
250ns seems too high, it's also higher than GH systems, is it expected?

Reply:
should it be 0 to avoid potential perf degradations on non GH systems?

Reply:
Yes, it should be 0 to keep the old behavior.